packages/cwp-template-aws/template/ddb/dependencies.json

```diff
@@ -50,7 +50,7 @@
     "typescript": "4.7.4"
   },
   "workspaces": {
-    "packages": ["apps/admin", "apps/website", "apps/api/graphql", "extensions/theme", "packages/*"]
+    "packages": ["apps/admin", "apps/website", "apps/api/graphql", "packages/*", "extensions/theme"]
```

Comment:
It should be `extensions/*`, just like `packages/*`, and then you never need to modify these rules again.

Reply:
The thing is, users can also create extensions in e.g. `extensions/fm/api/xyz` folder.

Reply:
If you think everyone will be using scaffolds, then probably this is ok.

Reply:
Yes, every extension article we'll have will essentially start with scaffold run. Examples:

https://docs-webiny-com-git-release-5400-webiny.vercel.app/docs/file-manager/extending/customize-file-preview
https://docs-webiny-com-git-release-5400-webiny.vercel.app/docs/file-manager/extending/customize-file-actions